cabby/geo/walk.py

```diff
@@ -36,10 +36,49 @@
 
 _Geo_DataFrame_Driver = "GPKG"
 OSM_CRS = 32633  # UTM Zones (North).
+SEED = 4
+POI_DISTANCE = 50
```

Comment:
Looks like two files use this module-level variable. Could it go in a module that both of them import? (if that is inconvenient, the way you have it is probably fine)

Reply:
this is removed